src/com/esotericsoftware/kryo/serializers/TaggedFieldSerializer.java

```diff
@@ -46,9 +49,12 @@
 	private int[] tags;
 	private int writeFieldCount;
 	private boolean[] deprecated;
+	/** If true, ignore unknown field tags. Otherwise throw KryoException. */
+	private boolean ignoreUnknownTags;

```

Comment:
Right, you can add a (maybe protected) Constructor to `FieldSerializer` which takes the `FieldSerializerConfig` separately, so that the existing `FieldSerializer` constructor can delegate to that one pulling the config from Kryo, and `TaggedFieldSerializer` can pass it's own config...